helm/fiftyone-teams-app/README.md.gotmpl

```diff
@@ -383,7 +382,6 @@ To configure this, set the following environment variables on
     > you will need to include the override service names in your
     > `GLOBAL_AGENT_NO_PROXY` configuration instead
 
-
```

Comment:
Removed extra unnecessary new lines that showed as warnings for markdownlint plugin